configs/mmdet/detection/single-stage_snpe_static.py

```diff
@@ -0,0 +1,4 @@
+_base_ = ['../_base_/base_static.py', '../../_base_/backends/snpe.py']
+
+codebase_config = dict(model_type='ncnn_end2end')
```

Comment:
do we support detection models in snpe? `ncnn_end2end` is a bit misleading.

Reply:
This file has been removed,  snpe not support detection model.